fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -1153,6 +1154,8 @@ def _reset_lazy_init(self) -> None:
         self._is_root: Optional[bool] = None
         self._streams: Dict[str, torch.cuda.Stream] = {}
         self._reducer: Optional[ReduceScatterBucketer] = None
+        self._fsdp_forward_ordering: List[nn.Module] = []
```

Comment:
We can remove this and all below changes not relevant to gradient_predivide_factor since FairScale already uses a 2 process group solution to implement overlap of BW compute with communication.